WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1 +1,5 @@
-Block editor: New block layout grid, Badge style tweaks, Fixed new block not created when pressing enter inside text-based block, Tablet view fixes for inserter button
+* [*] Fixed a bug where the web version of the editor did not load when using an account created before December 2018. [#14762]
+* [*] Block editor: Update loading and failed screens for web version of the editor [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3573]
+* [*] Block editor: Handle floating keyboard case - Fix issue with the block selector on iPad. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3687]
```

Comment:
This item was removed from the release notes submitted to Editorial after the fact once @thehenrybyrd noticed it shouldn't have made it to the Android notes – see p2JRYi-5q8-p2#comment-20679